nextflow.config

```diff
@@ -13,3 +13,26 @@ manifest {
   version = '0.0.1'
   defaultBranch = 'main'
 }
+
+profiles {
```

Comment:
I really like what you're thinking here, I should have been more specific and said that we would want to be able to specify different memory parameters per challenge and also between different nextflow tasks. Once we add more steps, you might imagine a combination of the following.

1. Specify these like nf-dcqc: https://github.com/Sage-Bionetworks-Workflows/nf-dcqc/blob/1e5f076905f4787bfc32b8229cfe4eb9f24194c3/modules/local/run_process.nf#L4
2. Specify these to the process like genie: https://github.com/Sage-Bionetworks/genie-bpc-pipeline/blob/382b57429e02873b11c3fd0871af753fe788d8a2/nextflow.config#L16

Reply:
in the nf-dcqc example, where do the variable memory and CPU values come from to get plugged in for `${memory}` and `${cpus}`?

Reply:
It is part of the input parameters as a tuple.